src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -461,6 +476,8 @@ private void SetupScene()
                 Position = new Point3D(10, 15, 10),
                 LookDirection = new Vector3D(-10, -10, -10),
                 UpDirection = new Vector3D(0, 1, 0),
+                NearPlaneDistance = 1,
+                FarPlaneDistance = 2000000,
```

Comment:
So, just to confirm, it looks this property defaults to positive infinity.  I assume from this we must set it.  That value just seems too small.  The pathological case is the stadium in mm units in Revit.

https://msdn.microsoft.com/en-us/library/system.windows.media.media3d.projectioncamera.farplanedistance(v=vs.110).aspx